doc/releases/release-notes-4.0.rst

```diff
@@ -1095,9 +1291,29 @@ Nanopb
 * Updated the nanopb module to version 0.4.9.
   Full release notes at https://github.com/nanopb/nanopb/blob/0.4.9/CHANGELOG.txt
 
+HOSTAP
+******
+
+  * Added DPP support for AP mode.
+  * Fix the issue of HE IE in beacon when disabled 11AX.
+  * Added full AP mode support based on hostapd
+  * Added 11k/v/r roaming support.
+  * Added WPA-Auto-Personal(WPA2/WPA3 mixed) support.
+  * Added AP configuration cmd support
+  * Fixed disconnect print fail log
+  * Expanded wpas_band_to_zephyr() to support eap secure modes.
+  * Added support for getting AP status
+  * Added support for sending 11v BTM request
+  * Improved security by adding enterprise support for STA mode
+  * Fixed UDP RX low throughput in 5GHz for AP mode
+  * Fixed device hang when start AP on wrong channel
+  * Added support for getting Wi-Fi version
```

Comment:
again, likely already covered in https://github.com/zephyrproject-rtos/zephyr/pull/81216

Reply:
@jukkar Your PR didn't add a HOSTAP section but instead captured some (or most of this) under WiFi. Is this what you want to do going forward?

Reply:
@dleach02 I do not see much benefit to split the release note even more to have `Hostap` topic as it is part of Wi-Fi anyway.